beamer/models/claim.py

```diff
@@ -17,6 +18,7 @@ def __init__(
     ) -> None:
         super().__init__()
         self._latest_claim_made = claim_made
+        self.challengers: defaultdict[Address, int] = defaultdict(int)
```

Comment:
Do we need the `defaultdict` here?

Reply:
since we add new stakes on top of challengers once a new `ClaimMade` event arrives, I used this to safe me the 

```
old_stake = 0
if challenger in self.challengers:
    old_stake = self.challengers[challenger]
self.challengers[challenger] = old_stake + extra_stake
```

Reply:
I see. I'm mostly scared that accessing this somewhere will add an incorrect challenger.

Reply:
Yes, this is one of those things where Python really is missing something.
I've commented that maybe this could be made private, which could help with this sort of thing.